lib/wallet/http.js

```diff
@@ -468,13 +468,16 @@ class HTTP extends Server {
 
       const options = {
         rate: valid.u64('rate'),
+        blocks: valid.u32('blocks'),
```

Comment:
what is `blocks` used for?

Reply:
For fee estimation (rate calc) if rate is not passed.